resources/testharness.js

```diff
@@ -1680,7 +1763,9 @@ policies and contribution forms [3].
         return clone;
     };
 
-    RemoteTest.prototype.cleanup = function() {};
+    RemoteTest.prototype.cleanup = function() {
+        this.done();
```

Comment:
Looks a bit weird, can you add a comment about why it's this way?

Reply:
Sure. Here's what I wrote:

>     * `RemoteTest` instances are objects which represent tests running in
>     * another realm. They do not define "cleanup" functions (if necessary,
>     * such functions are defined on the associated `Test` instance within the
>     * external realm). However, `RemoteTests` may have "done" callbacks (e.g.
>     * as attached by the `Tests` instance responsible for tracking the overall
>     * test status in the parent realm). The `cleanup` method delegates to
>     * `done` in order to ensure that such callbacks are invoked following the
>     * completion of the `RemoteTest`.

Reply:
Excellent!